test/parallel/test-dns-resolve-promises.js

```diff
@@ -9,7 +9,7 @@ const dnsPromises = require('dns').promises;
 
 // Stub cares to force an error so we can test DNS error code path.
 // Returning a negative value means the error corresponding to the postiive
-// value. (This is to distinguish system errors from Node.js errors.)
+// value. (This is the convention libuv uses for reporting errors.)
 cares.ChannelWrap.prototype.queryA = () => -EPERM;
```

Comment:
Since this test is already exposing internals we could use the map from the internal `uv` binding rather than the one from `constants`.

Reply:
Done!